ReactViewResources/Loader/Internal/ComponentWithMountCallback.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { PropsWithChildren, useState, useEffect } from "react";
+
+export interface IComponentWithMountCallbackProps {
+    mounted?(): void;
+}
+
+export function ComponentWithMountCallback({ mounted, children }: PropsWithChildren<IComponentWithMountCallbackProps>): JSX.Element | null {
+    const [isMounted, setIsMounted] = useState(false);
```

Comment:
i actually wanted this way because I was getting a crash stating that there was a render after the hydration
this is basically avoiding 1 render to fix that hydration issue - by postponing the render in a way

Reply:
i don get it... I stand with my opinion